src/components/AddCustModal.js

```diff
@@ -20,7 +22,8 @@ export class AddCustModal extends Component {
     //   // event.target.age.value
     //   event.target.address.value
     // );
-    fetch("https://localhost:44344/api/Customer1/Create", {
+
+    fetch("https://localhost:44344/Create", {
```

Comment:
Your api url should be "https://localhost:44344/api/Customer" for all CRUD operations of Customer (of course this needs to be fixed in the API-projet first)